### DISCUSSION TITLE
Scrum Prep Milestone #2 - First Week

### DISCUSSION THREAD

**Discussion by YanehRuiz** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#213] [Weekly Retrospective]
Weekly Development Update: First week of Milestone 2

Hello team!

Please provide a brief comment detailing what you accomplished this week, including any challenges faced or insights gained. Make sure to submit your updates by the deadline, which is before: Friday, September 27, 2024 @ 11am.

Thank you for your collaboration!

As for my collaboration:

**Comment by RaquelQuinones** [comment]
Wrote some task and I've been working with fixing the firebase problem I am having with my computer.

**Comment by Chosen-Juan1** [comment]
Made tasks involving the Figma design of the admin's communication interface page and it's implementation to the main branch.

**Comment by ghost** [comment]
I haven't had time to work on the project yet, because of other schoolwork. I had a test yesterday, and I have 4 tests and a quiz next week, as well as a lab report due.

**Comment by javier-rivera8** [comment]
I started working on my tasks, beginning with the design of the deletion report page to incorporate a report history.

**Comment by shenied-maldonado** [comment]
I opened some issues for myself & my team to handle. Other than that, it was a slow week for development.

**Comment by misaelmercado1** [comment]
Provided info on the task done on milestone 1 to team leader and updated new task on github for completion for milestone 2

**Comment by jonVargas1** [comment]
Worked on and implemented the report viewing UI

**Comment by Jjulianirizarry24** [comment]
This week, I had meetings and created multiple issues for team members to pick from to start working on their task according to the most critical goals at the moment. Have not had time to start working on completing tasks but next week will be more productive on that end

**Comment by lex939** [comment]
Several documentation tasks were uploaded so me and hybrids can start working on Milestone 2 and finish some tasks by the end of the first sprint.

**Comment by JorgeRivera94** [comment]
Edited the Report Creation wireframes, following manager feedback and UPR's identity manual. Started sampling of UPRM criminal acts reports.
Participated in team meeting where tasks and project logistics where discussed.

**Comment by noel-vargas** [comment]
Creates 2 new tasks, and linked my forked repo of the react components into the task

**Comment by Alejandro-DeJesus** [comment]
Conducted the weekly team meeting where tasks and project logistics where discussed. Haven't gotten around to working on any issues directly because of exams this week.

**Comment by NathBat09** [comment]
Discussed with my team the plan for this milestone.

**Comment by CaribbeanCool** [comment]
Participated in team meetings, helped finish the milestone 1 presentation for my team, and added functionality to the `Get Location` button

**Comment by yadrielsamir** [comment]
Conducted weekly meeting and started calendar design for admins.

**Comment by joel-alvarado** [comment]
Researched different methodologies for #190

**Comment by williammartinez10** [comment]
Didn't have the opportunity to work on the project. However, I’ve been considering new issues that could be created to improve the app's flow when a user initially opens it.

**Comment by evnd04** [comment]
This week i dindt have much time to work on the project but did attend all meetingsand approved most issues submited before yesterday.

**Comment by johnsantiago9** [comment]
Started in setting up environment in my computer.  My laptop was being fixed during milestone 1 and Ive been using a temp.

**Comment by JuanJRivera** [comment]
I was talking with my teammates in search of the tasks we will do for this Milestone. And research them.

**Comment by JorgeC08** [comment]
Working on the debugging the password handling firebase bug

**Comment by Rodal1** [comment]
We had a meeting and discussed some issues but I haven't done anything yet, next week is kinda tough for me I've got 3 exams. I'll be submitting issues by next week.
